### PR TITLE
Update runtime to 23.08

### PR DESCRIPTION
**Pull request by bztd**

### CODE REVIEW

**Comment by bztd** [comment]
@bbhtt Could you please update it? I have done partial tests and it works.

**Comment by bbhtt** [comment]
Can you please squash the commits ? And some indentation is broken in the manifest.

File: net.sourceforge.atanks.json [hunk 1]
```diff
@@ -43,31 +45,35 @@
...
+            "buildsystem": "autotools",
             "make-args": [
                 "PREFIX=/app"
             ],
             "make-install-args": [
                 "PREFIX=/app"
             ],
+	    "post-install": [
+		    "install -Dm644 atanks2.png /app/share/icons/hicolor/256x256/apps/atanks.png"
+	    ],
```
Comment (bbhtt) [inline net.sourceforge.atanks.json]: indent

File: net.sourceforge.atanks.json [hunk 2]
```diff
@@ -43,31 +45,35 @@
...
                     "type": "archive",
-                    "url": "https://downloads.sourceforge.net/project/atanks/atanks/atanks-6.5/atanks-6.5.tar.gz",
-                    "sha256": "32182b2752a77ff362c378fc04b7c51fc15345caa4c8deaad59cc850dad2322e"
+                    "url": "https://downloads.sourceforge.net/project/atanks/atanks/atanks-6.6/atanks-6.6.tar.gz",
+                    "sha256": "bc6b1efc9fc7e7624fad1d8352d72e927be0fba2160bb25bb6b8bd6c6370bf43"
                 },
+		{
+		    "type": "file",
+		     "path": "atanks2.png"
+		},
```
Comment (bbhtt) [inline net.sourceforge.atanks.json]: indent

**Comment by bbhtt** [comment]
Fixed up the commit history.

bot, build

**Pull request merged**